gha-webhook-listener.py

```diff
@@ -195,9 +200,14 @@ def deploy_tarball(artifact_url, target_dir):
     resp = requests.get(artifact_url, stream=True, headers=req_headers())
     resp.raise_for_status()
 
+    # GHA artifacts are wrapped in a zip file, so we extract it to get our tarball
+    # See https://github.com/actions/upload-artifact/issues/109
+    zipped_artifact = zipfile.ZipFile(BytesIO(resp.content))
+    tarball = zipped_artifact.open('content.tar.gz')
+
     # TODO: make the compression type depend on the filename
     # (or copy it to a temporary file so that tarfile can autodetect)
```

Comment:
we know what the filename is, because its hardcoded on the line above, so I think this TODO is redundant.